Units/parser-c.r/c-struct-var-with-initializer.d/expected.tags

```diff
@@ -0,0 +1,10 @@
+__anon31494def0108	input.c	/^struct {$/;"	struct	file:
+a	input.c	/^  int a;$/;"	member	struct:__anon31494def0108	typeref:typename:int	file:
```

Comment:
These anonymous identifiers come out different in my tests and the units fails. I wonder why...as the CMDLINE path seems to be relative?